plugin/src/main/java/io/jenkins/plugins/analysis/core/model/FilesScanner.java

```diff
@@ -106,10 +109,12 @@ private void aggregateIssuesOfFile(final Path file, final Report aggregatedRepor
 
             aggregatedReport.addAll(fileReport);
             aggregatedReport.setOriginReportFile(file.toString());
-            aggregatedReport.logInfo("Successfully parsed file %s", file);
-            aggregatedReport.logInfo("-> found %s (skipped %s)",
-                    plural(fileReport.getSize(), "issue"),
-                    plural(fileReport.getDuplicatesSize(), "duplicate"));
+            if (stdoutReport) {
+                aggregatedReport.logInfo("Successfully parsed file %s", file);
+                aggregatedReport.logInfo("-> found %s (skipped %s)",
+                        plural(fileReport.getSize(), "issue"),
+                        plural(fileReport.getDuplicatesSize(), "duplicate"));
```

Comment:
This is the wrong approach, there are several other places where the plugin writes log messages. The object oriented approach would be to add a quite flag here https://github.com/jenkinsci/warnings-ng-plugin/blob/master/plugin/src/main/java/io/jenkins/plugins/analysis/core/util/LogHandler.java.

This class writes all output to the console log.  

(Your code would also remove the logging messages from the info view of the plugin which obviously is not what you are trying to achieve.)

Reply:
1. Most likely I've missed it in docu, how verifying test case should look like - description/logs/snapshot/?
2. Thx for hint, will have a check on loghandler.java then.

Reply:
A simple unit test would be sufficient for the 2 cases:
- log handler default behavior
- log handler with system property set

Is there a reason why you are using a system property and not making the property visible as a step parameter?

Reply:
Ok, thx, will have a check on how-to unit-test here as well.
No reason, sys property was an initial idea - what is a `step param` in this context? The only goal is to calm jenkins job console output.

Reply:
With a system property you will shut up the whole instance. Normally you add options per invocation. 

E.g., a possible solution would create a new API in the style:

```
recordIssues quite: true, tools: [checkStyle(pattern: 'target/checkstyle-result.xml'),
                    spotBugs(pattern: 'target/spotbugsXml.xml'),
                    qualityGates: [[threshold: 1, type: 'TOTAL', unstable: true]]
```

Reply:
No issues with disk usage, the main reason is large console output which makes loading and reading the build logs a bit painy. We're checking about 1500logs x 4parsers per build. Its simply generating lots of lines with no added value, `found 0 issues (skipped 0 duplicates)` in most cases.

We wouldnt mind any other location like redirecting such output to other file or move to debug verbosity, /dev/null, just somewhere else. The overall results are nicely visible in separated warning subpages anyway.